Exercise/compute_lowest_terms.py

```diff
@@ -1,40 +1,73 @@
 # TODO: complete the function "lowest_terms" below
 
 def lowest_terms(x):
+	"""
+	Reducing fractions basically requires finding factors and dividing by them.
+	Containers (lists) are created e.g cont, factors and results to hold these
+	generated values.
+	"""
 	x = x.split('/')
-	l = []
-	fac = []
-	res = []
+	cont = []
```

Comment:
still unclear what 'cont' means (use better variable name)

Reply:
It's a container, I added that to the comment.